mesonbuild/coredata.py

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
```

Comment:
nit-picky, but there is a comma at the end of the commit message that doesn't belong. And while you're changing it, might throw the comma after the word "interpreter"